src/pages/CareerPage/CareerPage.js

```diff
@@ -1,17 +1,123 @@
 import React from "react";
 
 import CareerResourceList from "../../components/CareerResourceList/CareerResourceList";
+//import { Table } from "@material-ui/core";
```

Comment:
Shouldn’t need any commented out code, or do we?

Reply:
Deleted comments